samples/server/go-server/api/server/storage/queries.go

```diff
@@ -16,48 +16,48 @@ package storage
 
 const (
 	createTables = `
-		CREATE TABLE IF NOT EXISTS project (
+		CREATE TABLE IF NOT EXISTS projects (
 			id SERIAL PRIMARY KEY,
 			name TEXT NOT NULL UNIQUE
 		);
-		CREATE TABLE IF NOT EXISTS note (
```

Comment:
sorry I didn't get back to you sooner on this.

My suggestion is for each of these to separate the project and resource id so we'd have
                        id SERIAL PRIMARY KEY,
                        project_name TEXT NOT NULL UNIQUE
  			note_name TEXT NOT NULL UNIQUE,
  			data TEXT

Where we would interpret project name as only the project_name part of projects/**project_name**
This would allow us to change paths, if we want/need to in the future without storage issues.  There's been some conversation about whether notes should be under a provider path or remain under project path. 

Thoughts?

Reply:
Agree. This will be much better.

Reply:
Fixed in 90dbb92